<launch>
  <arg name="map_name" value="keti" />
  <arg name="config_file" default="$(arg map_name).yaml" />
  <!-- Odometry timestamp 추가 노드 -->
  <node pkg="epic_planner" type="odom_stamper.py" name="odom_stamper" output="screen"/>

  <include file="$(find epic_planner)/launch/algorithm.xml">
    <arg name="config_file" value="$(arg config_file)" />
  </include>
  
  <node pkg="plan_manage" name="traj_server" type="traj_server" output="screen">
    <remap from="/position_cmd" to="/planning/pos_cmd" />
  </node>
  

  
  <include file="$(find epic_planner)/launch/simulation_$(arg map_name).launch"/>
  
</launch>